tests/test_message.py

```diff
@@ -304,15 +304,31 @@ def test_chat_id(self, message):
         assert message.chat_id == message.chat.id
 
     def test_link(self, message):
```

Comment:
This unitest contains lot of internal changes of the `message` object.
Personally it makes me confused on what is the desired test.

Can you please split it into multiple unitests, each one testing one desired behaviour?